api/mapping/views.py

```diff
@@ -988,11 +1015,7 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
         x = ScanReport.objects.get(pk=self.kwargs.get("pk"))
-        context.update(
-            {
-                "scan_report": x,
-            }
-        )
+        context.update({"scan_report": x})
```

Comment:
odd that this doesnt fail the black job (?)

Reply:
The trailing comma at the end of line 993 was what caused it to spread out